prql/src/materializer.rs

```diff
@@ -127,8 +134,7 @@ impl RunFunctions {
             let value = func_call
                 .named_args
                 .iter()
-                // Quite inefficient — we could instead sort each list and join
-                // them. Is there a native way of doing that?
+                // Quite inefficient when number of arguments > 10
```

Comment:
Could we leave the note about how we could improve; or at least "we could improve the implementation with a merge join"?

Reply:
Sure